app/helpers/api/team.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DEFAULT_LOCALE} from '@i18n';
+
+export const selectDefaultTeam = (teams: Team[], locale = DEFAULT_LOCALE, userTeamOrderPreference = '', primaryTeam = '') => {
+    let defaultTeam;
```

Comment:
Do we also add unit test to helper functions ?

Reply:
Not yet.. this is part of the entry point logic that I'm working on.